hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/parse/SqlAlterJob.java

```diff
@@ -40,13 +43,17 @@ public class SqlAlterJob extends SqlAlter {
             new SqlSpecialOperator("ALTER JOB", SqlKind.OTHER_DDL);
 
     private final SqlIdentifier name;
+    private final SqlNodeList options;
     private final AlterJobOperation operation;
 
-    public SqlAlterJob(SqlIdentifier name, AlterJobOperation operation, SqlParserPos pos) {
+    private Map<String, Object> parsedOptions;
+
+    public SqlAlterJob(SqlIdentifier name, SqlNodeList options, AlterJobOperation operation, SqlParserPos pos) {
         super(pos, "JOB");
 
         this.name = requireNonNull(name, "Name must not be null");
-        this.operation = requireNonNull(operation, "Operation must not be null");
+        this.options = requireNonNull(options, "Options should not be null");
```

Comment:
not all `OPTIONS` + status combinations are supported. This is defined in SQL grammar, but could be also explicitly validated here: we support only `ALTER JOB .. OPTIONS (...) RESUME`